crates/libcontainer/src/workload/default.rs

```diff
@@ -57,52 +57,55 @@ impl Executor for DefaultExecutor {
             ))?;
 
         if let Some(args) = proc.args() {
-            let envs: Vec<String> = proc.env().as_ref().unwrap_or(&vec![]).clone();
-            let path_vars: Vec<&String> = envs.iter().filter(|&e| e.starts_with("PATH=")).collect();
-            if path_vars.is_empty() {
-                tracing::error!("PATH environment variable is not set");
-                Err(ExecutorValidationError::ArgValidationError(
-                    "PATH environment variable is not set".into(),
-                ))?;
-            }
-            let path_var = path_vars[0].trim_start_matches("PATH=");
-            match get_executable_path(&args[0], path_var) {
-                None => {
-                    tracing::error!(
-                        executable = ?args[0],
-                        "executable for container process not found in PATH",
-                    );
-                    Err(ExecutorValidationError::ArgValidationError(format!(
-                        "executable '{}' not found in $PATH",
-                        args[0]
-                    )))?;
+            if !args[0].contains('/') {
```

Comment:
What is the reason behind adding this check?

Reply:
Oh I shouldn't have included it in this PR. My reasoning for the change is that libcontainer shouldn't try to resolve the path for a program containing a `/` since it's likely an absolute path or relative path in which case it wouldn't be resolvable with the $PATH variable.

Reply:
Hey, this makes sense, but I'd request you to separate this change in its own PR, not mix with this one.